functionaltests/src/test/java/specs/BaseFixture.java

```diff
@@ -61,6 +59,7 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+import static com.commercetools.util.HttpRequestUtil.*;
```

Comment:
i'm not agree on these arguments: unless i have names conflicts i prefer to use wildcards, since they have less commit lines changes during re-factoring. 
This is default IDEA settings: if more then 3 items are imported from one namespace - use wildcards, thus there is no completely defined "community" conclusion of wildcards.

Reply:
> unless i have names conflicts
Right now you maybe don't. But in the future you could. It's a general good habit IMO.

> less commit lines to refactor
I don't think this affects in anyway does it?

Also, I don't think that being a "default IDEA settings" is a good argument :)

Reply:
> Also, I don't think that being a "default IDEA settings" is a good argument :)

I didn't see any strong arguments from your previous comments either. For me IDEA default settings are on the same level of authority as Stackoverflow questions.

Reply:
ok then I guess you don't need my review here..